ykman-gui/qml/Fido2ChangePinView.qml

```diff
@@ -26,7 +26,7 @@ ChangePinView {
                 if (resp.error === 'too long') {
                     fido2TooLongError.open()
                 } else if (resp.error === 'wrong pin') {
-                    clearPinInputs()
+                    clearCurrentPinInput()
```

Comment:
Testing this out, I think all PIN inputs should be cleared. Doing it like this allows the user to block the pin by hitting the Change PIN button repeatedly. Why the change?

Reply:
Hm, ok. It seemed reasonable to me to only reset the field that's wrong. I got annoyed that I had to re-enter all three values when I got the wrong current PIN, but I guess I've been doing that far more than you'd expect anyone would in normal use.

Would an alternative solution to the blocking the PIN problem be to keep the button disabled while the current PIN field is empty?

Reply:
> Would an alternative solution to the blocking the PIN problem be to keep the button disabled while the current PIN field is empty?

Yes it might, at least it makes sense to do anyway. Current PIN should have a minimum of 4 characters to enable the button.

Reply:
Let's try that solution and see how it feels.

Reply:
Done.